modules/manage/partials/iceberg/use-iceberg-catalogs.adoc

```diff
@@ -39,9 +38,114 @@ For REST catalogs that use self-signed certificates, also configure these proper
 
 See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
```

Comment:
we should probably hide the _file properties from cloud docs, as they cannot be applied. I did not go over this line by line but we should never reference anything else than what's on this list https://github.com/redpanda-data/cloudv2/blob/main/install-pack/25.1.yml#L80

Reply:
Updated so that the _file properties are only mentioned in the Self-managed version of the page